src/aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -426,13 +428,19 @@ async def do_kill():
             transport = await cancellable.with_interrupt(request)
             return execmanager.kill_calculation(node, transport)
 
+    if force_kill:
+        logger.warning(f'Process<{node.pk}> has been force killed! this may result in orphaned jobs.')
```

Comment:
Not sure if the language "orphaned jobs" may be immediately clear. I would perhaps clarify it a bit by saying something like:

> "`CalcJob<node.pk>` is being force killed. The associated job on the computer is not being killed through the scheduler and may continue running."

Just to make clear it is the job with the scheduler on the computer that is not explicitly killed.

Reply:
Thanks, indeed it's more clear now.